README.md

```diff
@@ -52,6 +52,12 @@ a sciml__v20240817__textembedding3large-1024-Bool__v1.0.hdf5
 ```
 `make_knowledge_packs` is the entry point to the package. This function takes in the URLs to parse and returns the index. This index can be passed to AIHelpMe.jl to answer queries on the built knowledge packs.
 
+**Default `make_knowledge_packs` Parameters:** 
```

Comment:
Looks good but one warning! default AIHelpMe pipeline is 1024dims, bool=true; The challenge is that to run with 3072dims and Float32, you need to enable `update_pipeline(:gold)` but that also adds a reranker that not everyone uses!

I would recommend:
- Keep default to how we build the knowledge packs (since that's a primary purpose of the package) (please make the kwarg names consistent!)
- In your usage examples, suggest to users that for everyday use they should use 1024-bool (sufficient) and make sure to use `update_pipeline(:bronze)` or `:silver` in AIHelpMe to make sure it's consistent and it works!
If you wanted you can suggest 3072-bool for higher performance, by setting `update_pipeline(:bronze; embedding_dimension=3072)` -- I'll leave the choice to you but please define it explicitly in the README to make sure we avoid these errors for users